src/components/category/catGroup.jsx

```diff
@@ -1,20 +1,37 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
+import { Link } from 'react-router-dom';
+import slugify from 'slugify';
+
 import CatItem from './catItem';
 
 
 const catGroup = (props) => {
   const items = props.items.map((item) => {
     if (!item.name) return false;
+
+    const buildSlug = string => slugify(string, {
```

Comment:
i see this code is duplicated, so would be better to extract this into a separate file and impor it in both files that use this function, they look pretty much the same